Scripts/freight_lem.py

```diff
@@ -86,6 +90,15 @@ def main(args):
             matrix_counter += 7 # Shift to next 10 digit
         ass_model.freight_network.output_traversal_matrix(resultdata.path)
         aux_demand = transform_traversal_data(resultdata.path, zone_numbers)
```

Comment:
alternatively we could simply add aux demand into demand["truck"] on return, and then use that matrix in calc_vehicles().